demos/buildah-scratch-demo.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash 
+
+# author : ipbabble
+# Assumptions install buildah and podman
+# login to Quay.io using podman if you want to see the image push
+#   otherwise it will just fail the last step and no biggy.
+#   podman login quay.io
+# Set some of the variables below
+
+demoimg=myshdemo
+quayuser=ipbabble
+myname=WilliamHenry
+fedorarelease=28
+
+#Setting up some colors for helping read the dmeo output
+bold=$(tput bold)
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+blue=$(tput setaf 4)
+cyan=$(tput setaf 6)
+reset=$(tput sgr0)
+
+echo -e "Using ${green}GREEN${reset} to introduce Buildah steps"
+echo -e "Using ${yellow}YELLOW${reset} to introduce code"
+echo -e "Using ${blue}BLUE${reset} to introduce Podman steps"
+echo -e "Using ${cyan}CYAN${reset} to introduce bash commands"
+echo -e "Building an image called ${demoimg}"
+read -p "${green}Start of the script${reset}"
+
+set -x
+read -p "${green}Create a new container on disk from scratch${reset}"
+newcontainer=$(buildah from scratch)
+read -p "${green}Mount the root directory of the new scratch container${reset}"
+scratchmnt=$(buildah mount $newcontainer)
+read -p "${cyan}Lets see what is in scratchmnt${reset}"
+ls $scratchmnt
+echo -e "${red}Note that the root of the scratch container is EMPTY!${reset}"
+read -p "${cyan}Time to install some basic bash capabilities: coreutils and bash packages${reset}"
+dnf install --installroot $scratchmnt --release ${fedorarelease} bash coreutils --setopt install_weak_deps=false -y
```

Comment:
yum here instead of dnf.

Reply:
I have added support for both yum or dnf. You jusy need to change a pkgmgr variable once.
Someone can add apt later :) Maybe me.